### PR TITLE
header images, simple footer, related styles

### PR DESCRIPTION
**Pull request by BryanNilsen**
created img directory and added header images for each search page, inserted header and footer into index.html, and added header/footer styles to main.css

### CODE REVIEW

**Comment by braddavistech** [comment]
👍 Looks good. I just ran through the site and saw that I need to add a remove function for all of the sections when we save the itinerary. Not a big deal and we can handle Monday morning.

File: src/styles/main.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/* remove default margin/padding */
+* {
```
Comment (scivarolo) [inline src/styles/main.css]: Are you just trying to remove the margin/padding around the window? I'd change this from `*` to `body` so it doesn't affect any margins and paddings elsewhere.

Comment (scivarolo) [inline src/styles/main.css]: Are you just trying to remove the margin/padding around the window? I'd change this from `*` to `body` so it doesn't affect any margins and paddings elsewhere.

Comment (BryanNilsen) [reply src/styles/main.css]: Yes, trying to remove the window margin/padding. If you think applying it to the body is better, I’m cool with that.

**Comment by scivarolo** [comment]
😎 🐝 
Looks good!

**Pull request merged**